ballerina/client.bal

```diff
@@ -37,14 +37,14 @@ public isolated client class Client {
     # + salesforceConfig - Salesforce Connector configuration
     # + return - `sfdc:Error` on failure of initialization or else `()`
     public isolated function init(ConnectionConfig config) returns error? {
-        http:Client|http:ClientError|error httpClientResult;
+        http:Client|http:ClientError httpClientResult;
         http:ClientConfiguration httpClientConfig = check config:constructHTTPClientConfig(config);
-        httpClientResult = trap new (config.baseUrl, httpClientConfig);
```

Comment:
Will it handle panic situations properly?

Reply:
@TharmiganK , Do we need to handle ```panics``` here ?

Reply:
This is just the HTTP client initialisation right? In error situations, we explicitly return a `http:ClientError` rather than panicing